### PR TITLE
✨ Add Input Field Component

### LINKED ISSUE #10: Input Field Component
### Screenshots

Bildschirmfoto 2021-06-13 um 01 15 35

### Tasks

- [x] Icon on the left
- [x] Placeholder text on the right

### PR DESCRIPTION
**Pull request by dhnazarenko**
fixes #10 

- Add input field component
- Add icons
- Change color in root
- Style component 

Check it out:

https://timeonation-input-mweq7elbrhz9.herokuapp.com/storybook/?path=/story/component-labeledinput--username

### CODE REVIEW

File: src/app/components/LabeledInput/LabeledInput.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  title: 'Component/LabeledInput',
+  component: LabeledInput,
+};
+
+export const username = (): JSX.Element => (
+  <LabeledInput
+    icon={<UserIcon />}
+    placeholder="Benutzername"
+    value=""
+    required={true}
```
Comment (lmachens) [inline src/app/components/LabeledInput/LabeledInput.stories.tsx]: There is a short form if you like (only for boolean) 
```suggestion
    required
```

**Review by lmachens — APPROVED** [review]
Beautiful

File: src/app/components/Icons/LocationIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function LocationIcon(): JSX.Element {
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" width="42" height="42">
+      <path
+        d="M572-159a20.87,20.87,0,0,1-8.174-1.65,20.93,20.93,0,0,1-6.675-4.5,20.929,20.929,0,0,1-4.5-6.675A20.868,20.868,0,0,1,551-180a20.868,20.868,0,0,1,1.65-8.174,20.929,20.929,0,0,1,4.5-6.675,20.93,20.93,0,0,1,6.675-4.5A20.871,20.871,0,0,1,572-201a20.868,20.868,0,0,1,8.174,1.65,20.929,20.929,0,0,1,6.675,4.5,20.929,20.929,0,0,1,4.5,6.675A20.868,20.868,0,0,1,593-180a20.868,20.868,0,0,1-1.65,8.174,20.929,20.929,0,0,1-4.5,6.675,20.929,20.929,0,0,1-6.675,4.5A20.868,20.868,0,0,1,572-159Zm-.055-31a6.9,6.9,0,0,0-4.912,2.033A6.9,6.9,0,0,0,565-183.055c0,2.691,1.888,6.055,3.473,8.4a45.335,45.335,0,0,0,3.473,4.5,45.807,45.807,0,0,0,3.473-4.5c1.584-2.348,3.473-5.712,3.473-8.4a6.9,6.9,0,0,0-2.033-4.912A6.9,6.9,0,0,0,571.945-190Zm0,9.427a2.484,2.484,0,0,1-2.48-2.481,2.483,2.483,0,0,1,2.48-2.48,2.483,2.483,0,0,1,2.48,2.48A2.484,2.484,0,0,1,571.945-180.573Z"
+        transform="translate(-551 201)"
+        fill="#269396"
```
Comment (Dev-dfm) [inline src/app/components/Icons/LocationIcon.tsx]:
```suggestion
        fill="currentColor"
```
like on the other icons

**Review by Dev-dfm — APPROVED** [review]
Looks great 🚀

File: src/app/components/Icons/LocationIcon.tsx (hunk 2 above)
Comment (dhnazarenko) [reply src/app/components/Icons/LocationIcon.tsx]: Thanks! I did not see that.

File: src/app/components/LabeledInput/LabeledInput.stories.tsx (hunk 1 above)
Comment (dhnazarenko) [reply src/app/components/LabeledInput/LabeledInput.stories.tsx]: Thanks! I changed it.

**Review by l-goede — APPROVED** [review]
lgtm!:)

**Pull request merged**